mantidimaging/gui/windows/image_load_dialog/test/presenter_test.py

```diff
@@ -233,77 +233,68 @@ def test_ensure_sample_log_consistency_exits_when_none_or_empty_str(self, mock_l
         mock_load_log.assert_not_called()
         mock_log.raise_if_angle_missing.assert_not_called()
 
-    @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.get_prefix", return_value="/path")
-    def test_get_parameters(self, get_prefix):
-        path_text = "/path/text"
-        sample_input_path = "/sample/input/path"
-        image_format = ".tif"
-        sample_indices = [1, 1]
-        sample_file_name = "/path/sample_file_name"
+    @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.FilenameGroup.find_all_files")
+    def test_get_parameters(self, _):
+        sample_path = "/sample/tomo/tomo_0001.tiff"
+        sample_file = "tomo_0001.tiff"
+        sample_log_path = "/sample/tomo.log"
+
+        self.fields["Sample"].path_text.return_value = sample_path
+        self.fields["Sample"].file.return_value = sample_file
+        self.fields["Sample"].use.isChecked.return_value = True
+        self.fields["Sample Log"].path_text.return_value = sample_log_path
+
+        # sample_indices = [1, 1] #TODO
```

Comment:
Did you mean to leave this in?

Reply:
It gets fixed in the next PR #1728 ( ba2b7bb6d83889c53a268534796fc9aef215a19e )